File: Java/Practice/Patterns/number8.java
// 1
// 2 3 4
// 5 6 7 8 9

public class number8 {

  public static void main(String[] args) {
    int num = 1;
    int count = 1;
    for (int i = 0; i < 3; i++) {
      for (int j = 1; j < count; j++) {
        System.out.print(num);
        num += 1;
      }
      System.out.println(" ");
      count += 2;
    }
  }
}

File: 03_Python/Others/Patterns/numbers/number8.py

# 1 
# 2 3 4 
# 5 6 7 8 9

n =3
s = 2
num = 1

for i in range(0,n):
    for j in range(1,s):
        print(num,end=" ")
        num+=1
    s +=2
    print("\r")